diagnostic.md

```diff
@@ -7,36 +7,39 @@ What command do we use to do that? Use the **correct** URL from your fork of
 this repository in your answer.
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+git clone git@github.com:alexkarasik/git-basics-diagnostic.git
 ```
 
 2.What do you do after cloning a repository, before you start making any
 changes/additions?
+You go into that directory that you just cloned.
 
-<!-- Remove this comment and place your answer here. -->
 
 3.What command do we use to create a new branch? Name this branch `response`
     in your answer. Then, how do we switch to that branch?
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+You give the command git branch <branchname>.
+Then git checkout <branchname>.
+We can combine these steps with git checkout -b <branchname>.
+
 ```
 
 4.We just wrote some code. What command do we use to see a summary of the
     changes in our working directory?
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+git diff will show us a summary of the changes we make.
 ```
 
 5.We want to prepare a change for a commit by adding a file to the staging
     area. What command do we use to stage a file named `diagnostic.md`?
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+git add diagnostic.md
 ```
 
 6.Once `diagnostic.md` is staged, we have to make a commit by `git commit`.
 What are the two formatting items you **need** to make up your commit message?
 
-<!-- Remove this comment and place your answer here. -->
+ Using the imperative to describe the issue along with explaining what and why you made certain changes rather than how.
```

Comment:
The two formatting items that we're looking for in a good commit message are a short summary title and a body detailing the changes made.